packages/app-desktop/gui/ConversionNotification/ConversionNotification.tsx

```diff
@@ -1,28 +1,31 @@
 import * as React from 'react';
-import { useContext, useEffect } from 'react';
+import { useEffect } from 'react';
 import { _ } from '@joplin/lib/locale';
 import { Dispatch } from 'redux';
-import { PopupNotificationContext } from '../PopupNotification/PopupNotificationProvider';
 import { NotificationType } from '../PopupNotification/types';
+import useToastNotifier from '../../gui/hooks/useToast';
```

Comment:
Either you call it useToastNotifier or you call it useToast, but you need to pick one as we can't have both. Second time I'm giving you a link to the coding style which you should have read:

https://joplinapp.org/help/dev/coding_style/#use-the-same-case-for-imported-and-exported-members